app/views/employees/_form.html.erb

```diff
@@ -1,5 +1,11 @@
 <%= simple_form_for @employee do |form| %>
-  <%= form.input :slack_username %>
+  <%= form.input :slack_username,
+                 required: true,
+                 pattern: '^[^@]\w+[^ ]$',
+                 input_html: {
+                   title: I18n.t('employees.errors.slack_username')
```

Comment:
I implemented my idea in 86738f9. I was also wondering if we need the validation on [line 16](https://github.com/18F/dolores-landingham-bot/blob/master/app/controllers/employees_controller.rb#L16). Perhaps there is an edge case I am not grasping but it seems unnecessary to reject users that are already in your organization if they fail to save. I think the check I added in my last commit should handle any dangling edge cases. Thoughts?